tools/run_mview_sperson_estimator.py

```diff
@@ -0,0 +1,217 @@
+# yapf: disable
+import argparse
+import glob
+import mmcv
+import os
+from xrprimer.data_structure.camera import (
+    FisheyeCameraParameter, PinholeCameraParameter,
+)
+from xrprimer.utils.log_utils import logging, setup_logger
+from xrprimer.utils.path_utils import Existence, check_path_existence
+from xrprimer.visualization.keypoints.visualize_keypoints3d import (
+    visualize_keypoints3d,
+)
+
+from xrmocap.core.estimation.builder import build_estimator
+from xrmocap.data_structure.body_model import SMPLXData
+from xrmocap.utils.date_utils import get_datetime_local, get_str_from_datetime
+from xrmocap.visualization.visualize_keypoints3d import (
+    visualize_keypoints3d_projected,
+)
+from xrmocap.visualization.visualize_smpl import visualize_smpl_data
+
+# yapf: enable
+
+
+def main(args):
+    filename = os.path.basename(__file__).split('.')[0]
+    # check input and output path
+    logger = setup_logger(logger_name=filename)
+    if len(args.data_root) <= 0 or \
```

Comment:
```suggestion
    if os.path.exists(args.data_root) or \
```

Reply:
Fixed in [83ac66b](https://github.com/openxrlab/xrmocap/pull/123/commits/83ac66b600c1681e720382df169657ac87ed3b88).